rqt_top/src/rqt_top/top_plugin.py

```diff
@@ -34,6 +34,8 @@
 from python_qt_binding.QtCore import Qt, QTimer
 
 from rqt_top.node_info import NodeInfo
+from rqt_top.msg import *
```

Comment:
If I have one pet peeve about python style, it's `import *`. Please explicitly import what you need.